src/server/utils.js

```diff
@@ -263,7 +271,7 @@ window.LiveReloadOptions = {
     ctx.log[level](`Proxy ${req.method} request to ${url}: ${ret.status} (${contentType})`);
 
     const isHTML = ret.status === 200 && contentType.indexOf('text/html') === 0;
-    const injectLR = isHTML && opts.injectLiveReload;
+    const livereload = isHTML && opts.injectLiveReload;
```

Comment:
the `port` should be part of the hostname

Reply:
I'm not sure what you are suggesting. :(

Reply:
sorry. must have looked at an outdated change. don't know why I wrote that :-)